bin/metrics-cloudfront.rb

```diff
@@ -128,18 +128,34 @@ def run
       '5xxErrorRate' => 'Average'
     }
 
-    unless config[:metric].nil?
-      statistic.select! { |key, _| key == config[:metric] }
+    metrics.each do |metric|
+      statistic = metrics_statistic.select { |key, _| key == metric }
+      if statistic.empty?
+        unknown "Invalid metric #{metric}. Possible values: #{metrics_statistic.keys.join(',')}"
+      end
+      print_statistics(distribution_id, statistic)
+    end
+  end
+
+  def parse_metrics(metrics)
+    if metrics.nil?
+      unknown 'No metrics provided. See usage for details'
     end
+    ret = metrics.split(',')
```

Comment:
the last action completed in ruby is implicitly returned in a function.
```suggestion
    metrics.split(',')
```